DependencyInjection/Configuration.php

```diff
@@ -131,12 +161,13 @@ private function createPluginsNode()
      */
     private function createBindNode()
     {
-        return $this->createNode('bind')
-            ->useAttributeAsKey('name')
-                ->prototype('array')
+        return $this
+            ->createNode('bind')
                 ->useAttributeAsKey('name')
-                ->prototype('variable')->end()
-            ->end();
+                ->prototype('array')
+                    ->useAttributeAsKey('name')
+                    ->prototype('variable')->end()
+                ->end();
```

Comment:
Aiming to make the following configuration https://github.com/nao-pon/elFinder/blob/2.1_n/php/plugins/AutoResize/plugin.php#L9